OJP_Locations.xsd

```diff
@@ -343,6 +341,11 @@
 					<xs:documentation>Rough estimate of the travel duration from the specified location to this exchange point.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="WaitDuration" type="xs:duration" minOccurs="0">
```

Comment:
Would it make sense to add a `default="0"`?

Reply:
Added default PT0M into new commit.